pkg/partials/partial.go

```diff
@@ -24,36 +25,40 @@ func GetRequestType(payloadBz []byte) (sharedtypes.RPCType, error) {
 
 // GetErrorReply returns an error reply for the given payload and error,
 // in the correct format required by the request type.
-func GetErrorReply(payloadBz []byte, err error) ([]byte, error) {
-	partialRequest, er := PartiallyUnmarshalRequest(payloadBz)
+func GetErrorReply(ctx context.Context, payloadBz []byte, err error) ([]byte, error) {
```

Comment:
It's cool that a result of having a structured logger forced us to propagate the context throughout!

Reply:
:100: